src/main/kotlin/team/msg/hiv2/domain/notice/persistence/NoticePersistenceAdapter.kt

```diff
@@ -0,0 +1,33 @@
+package team.msg.hiv2.domain.notice.persistence
+
+import org.springframework.data.repository.findByIdOrNull
+import org.springframework.stereotype.Component
+import team.msg.hiv2.domain.notice.application.spi.NoticePort
+import team.msg.hiv2.domain.notice.domain.Notice
+import team.msg.hiv2.domain.notice.exception.NoticeNotFoundException
+import team.msg.hiv2.domain.notice.persistence.mapper.NoticeMapper
+import team.msg.hiv2.domain.notice.persistence.repository.NoticeRepository
+import java.util.*
+
+@Component
+class NoticePersistenceAdapter(
+    private val noticeMapper: NoticeMapper,
+    private val noticeRepository: NoticeRepository
+) : NoticePort {
+
+    override fun saveNotice(notice: Notice) {
+        noticeRepository.save(noticeMapper.toEntity(notice))
+    }
+
+    override fun deleteNotice(noticeId: UUID) {
+        noticeRepository.deleteById(noticeId)
+    }
+
+    override fun queryNotice(): List<Notice?> {
+        val notice = noticeRepository.findAll()
+        return notice.map { noticeMapper.toDomain(it) }
+    }
```

Comment:
```suggestion
    override fun queryNotice(): List<Notice?>  =
        return noticeRepository.findAll().map { noticeMapper.toDomain(it) }
```
이렇게 더 간결하게 표기하는건 어때요? 일단 findAll로 받는 변수이름이 notices도 아니고 noticeList도 아니고 notice 인것도 이상하긴하네요